packages/sanes-browser-extension/src/routes/tx-request/components/ProposalActionType/ReleaseEscrow.tsx

```diff
@@ -0,0 +1,48 @@
+import { ReleaseEscrowAction } from "@iov/bns";
+import { ListItem, ListItemText } from "medulas-react-components";
+import * as React from "react";
+import { amountToString } from "ui-logic";
+
+import {
+  txListItemSecondaryProps,
+  useTxListItemHeaderStyles,
+  useTxListItemStyles,
+} from "../ShowRequest/TransactionFee";
+
+interface Props {
+  readonly header?: boolean;
+  readonly action: ReleaseEscrowAction;
+}
+
+function ReleaseEscrow({ action, header }: Props): JSX.Element {
+  const listItemClasses = useTxListItemStyles();
+  const listItemHeaderClasses = useTxListItemHeaderStyles();
+
+  return (
+    <React.Fragment>
+      {header && (
+        <ListItem>
+          <ListItemText classes={listItemHeaderClasses} primary="Release Escrow" />
+        </ListItem>
+      )}
+      <ListItem>
+        <ListItemText
+          classes={listItemClasses}
+          primary="Escrow ID"
+          secondary={action.escrowId}
```

Comment:
So should I change the code, because this property will be changed to `number`? Also [same issue](https://github.com/iov-one/ponferrada/blob/0baf8769880f36f52e394611598aebde913fd3f9/packages/sil-governance/src/routes/dashboard/components/Proposal/CollapsingData/DisplayOptions/ReleaseEscrow.tsx#L14).

Reply:
Yes, please change. Then we get the future look and feel already and can replace `Uint64.fromBytesBigEndian(action.escrowId).toNumber()` with just `action.escrowId` once IOV-Core 1.0.0 is released

Reply:
This is how it looks with long number and Uint8Array directly. 
![release-escrow](https://user-images.githubusercontent.com/3502260/65234207-d2742c80-dadc-11e9-9964-66adb7b0aab8.png)

Reply:
This representation looks very random for the input data `0434ce248a6a5979c04d75d1a75907b2bec1cb4d4f6e17b76521f0925e8b6b40e00711fe98e789cf5c8317cf1e731b3101e9dbfaba5e351e424e45c9a2f4dfb63c`. Maybe it is a list of numeric bytes separated byt the empty string. However, it does not look very helpful.

Reply:
Yeah,  I found this too